driver-core/src/test/java/com/datastax/driver/core/CCMBridge.java

```diff
@@ -144,7 +148,7 @@
     private static final String CCM_COMMAND;
 
     static {
-        CASSANDRA_VERSION = System.getProperty("cassandra.version");
+        INPUT_CASSANDRA_VERSION = System.getProperty("cassandra.version");
```

Comment:
I didn't want to change the existing system properties, I'll think on this for a bit